src/chart/sankey/SankeySeries.js

```diff
@@ -138,6 +138,17 @@ var SankeySeries = SeriesModel.extend({
         }
     },
 
+    // Override Series.getDataParams()
+    getDataParams: function (dataIndex, dataType) {
+        var params = SankeySeries.superCall(this, 'getDataParams', dataIndex, dataType);
+        if (!params.value && dataType === 'node') {
```

Comment:
Should use
```js
if (params.value != null && dataType === 'node')
```
To avoid `0` value can't enter this condition issue.

Reply:
OK, thanks. Frankly, I forget the special value `0`. But I think it should be:
```js
if (params.value == null && dataType === 'node')
```
because vlaue 0 shoule not be override